features/step_definitions/amazon_search.rb

```diff
@@ -0,0 +1,22 @@
+Given("I am on Amazon home page") do
+  navigate_to($app_path)
+end
+
+When("I enter the text {string} on the top search bar") do |search|
+  top_search_bar = $driver.find_element(id: 'twotabsearchtextbox')
+  top_search_bar.send_keys search
```

Comment:
Yo can use `enter_text(typeIdentifier, identifier, text)` to avoid "find_element" and "send_keys"